virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-supported-features-for-virtual-storage"]
+= Supported features for virtual storage
+include::modules/virt-document-attributes.adoc[]
+:context: virt-supported-features-for-virtual-storage
+toc::[]
+
+Use the following table to determine feature support for local and shared
+persistent virtual storage in {VirtProductName}.
```

Comment:
I recommend removing the word virtual as you are referring to real storage providers (unless you are referring to virtual disks in which case you should add that word).

Reply:
Changed "virtual storage" to "storage". Will rename modules and assembly to reflect this change also.